client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -30,119 +30,118 @@
       <i
         class="fa fa-bars handle w-[20px] cursor-grab"
         :title="Translator.trans('move')" />
-    </div>
+    </div><!--
+ --><div
+    class="inline-block layout--flush c-at-item__row"
+    data-cy="mapLayerListItem">
     <div
-      class="flex w-full"
-      data-cy="mapLayerListItem">
-      <div class="flex-1">
-        <!-- regular categories -->
-        <i
-          v-if="layer.type === 'GisLayerCategory' && false === layer.attributes.layerWithChildrenHidden"
-          aria-hidden="true"
-          class="fa u-mr-0_125"
-          @click="toggleChildren"
-          :class="[childElements.length > 0 ? (showChildren ? 'fa-folder-open' : 'fa-folder') :'fa-folder-o']" />
-        {{ layer.attributes.name }}
-        <span
-          class="font-size-smaller u-mr-0_5"
-          v-if="isChildOfCategoryThatAppearsAsLayer && 'mapOrder' === sortingType">
+      class="inline-block w-9/12"
+      :class="{ 'w-11/12': !hasPermission('feature_map_layer_visibility') }">
+      <!-- regular categories -->
+      <i
```

Comment:
I think a native interactive element (like a button) would be beneficial here to improve accessibility. Then an icon could be added which could have the `aria-hidden=true` attribute